server/src/routes/api/v1/usersRouter.js

```diff
@@ -2,11 +2,24 @@ import express from "express";
 import passport from "passport";
 import { User } from "../../../models/index.js";
 import objection from "objection";
+import UserSerializer from "../../../serializers/UserSerializer.js";
 
 const { ValidationError } = objection;
 
 const usersRouter = new express.Router();
 
+usersRouter.get("/", async (req, res) => {
```

Comment:
Then here, we would make this a GET endpoint for a single user based on their `/:id`. You could even verify that the `req.user.id` is the same as the `req.params.id`